### PR TITLE
Update broken links

### PR DESCRIPTION
**Pull request by emmanuel-ferdman**
### Description
Commit a16f11ad6ee01389bdb68ee499b85c676e283d36 moved bunch of files. This PR adjusts sources to changes.

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Closes # .

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1.
2.
3.

### Changelog entry

-   [ ] Automatically create a changelog entry from the details below.

-   [ ] This Pull Request does not require a changelog entry. (Comment required below)

<details>

<summary>Changelog Entry Details</summary>

#### Significance

-   [ ] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>

### CODE REVIEW

File: docs/extension-development/handling-merchant-onboarding.md [hunk 1]
```diff
@@ -287,7 +287,7 @@ By following these steps, your custom task should appear in the WooCommerce onbo
 
 For a complete example of adding a custom task as a WordPress plugin, you can check out the [add-task examples directory](https://github.com/woocommerce/woocommerce/tree/trunk/plugins/woocommerce-admin/docs/examples/extensions/add-task).
```
Comment (lanej0) [inline docs/extension-development/handling-merchant-onboarding.md]: Think you missed one here!

**Review by lanej0 — CHANGES_REQUESTED** [review]
Looks good.  Just one missed link that I noticed in the review.

**Comment by emmanuel-ferdman** [comment]
@lanej0 thanks for the review, I fixed it

**Comment by lanej0** [comment]
Looks good!  I kicked off the checks and it looks like there was just one failure in the Markdown linting (it may even be un-related to your changes, but if you wouldn't mind taking a look/fixing it along with this PR, that would be great).  Also just need a changelog entry with this.  If you could run `pnpm --filter='@woocommerce/plugin-woocommerce' changelog add` and fill in the details, we should be able to get this merged!  Thanks for your contribution.

**Review by lanej0 — APPROVED** [review]

**Comment by emmanuel-ferdman** [comment]
@lanej0 fixed the markdown errors and added the changelog entry

**Comment by lanej0** [comment]
I'm really sorry, it's been a while since I did docs updates and there's one more step.  Can you run `pnpm utils md-docs create docs woocommerce -o docs/docs-manifest.json` and commit the file that's changed.  That will update the manifest and get the lint check passing.

**Comment by emmanuel-ferdman** [comment]
@lanej0 added, please merge

**Pull request merged**